hudi-common/src/main/java/org/apache/hudi/avro/HoodieAvroUtils.java

```diff
@@ -214,6 +218,20 @@ private static GenericRecord rewrite(GenericRecord record, Schema schemaWithFiel
     return newRecord;
   }
 
+  /*
+  This function takes the union of all the fields except hoodie metadata fields
+   */
+  private static List<Field> getAllFieldsToWrite(Schema oldSchema, Schema newSchema) {
```

Comment:
in that case, let have the name reflect that.. `getCombinedFields()`.. `All` does not bring out what this is doing IMO.

Reply:
Changed it to getCombinedFieldsToWrite()